data/pirate slavery jobs.txt

```diff
@@ -0,0 +1,1073 @@
+# Copyright (c) 2014 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "Slave Transport [0]: Release Slaves"
+	minor
+	invisible
+	repeat
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral"
+	to offer
+		has "Slave Transport [0]: active"
+		"sold slaves" > 0
+		or
+			"reputation: Republic" > 0
+			"reputation: Syndicate" > 0
+			"reputation: Free Worlds" > 0
+	on offer
+		conversation
+			`Luckily, the security on <planet> did not detect the slaves on your ship. You've sold your passengers into slavery before, so maybe you could release these slaves to the local government and recieve a reward.`
+			choice
+				`	(Accept.)`
+				`	(Decline.)`
```

Comment:
I don't think "accept" and "decline" apply here. Maybe say "Free the slaves" or "Keep the slaves."